server/src/main/java/io/spine/server/entity/storage/ColumnRecords.java

```diff
@@ -32,6 +34,8 @@
  * A utility for dealing with the {@linkplain EntityRecordWithColumns} and
  * the {@linkplain EntityColumn entity columns}.
  */
+@Experimental
+@Internal
```

Comment:
It is at least `SPI`. This utility is required by the storage implementations.

Reply:
Same as above. We don't want SPI-users to rely on it at this point.